.github/workflows/ci.yaml

```diff
@@ -107,4 +100,4 @@ jobs:
       with:
         token: ${{ secrets.CODECOV_TOKEN }}
         file: ./coverage.xml
-        fail_ci_if_error: true
+        fail_ci_if_error: false  # I don't want this to cause CI failures when a developer pushes to a fork
```

Comment:
```
      if: ${{ github.repository == 'openmm/openmmforcefields'
              && github.event != 'schedule' }}
```

I thought this logic would prevent that, but if it wasn't working then this change is great for the same reasons we got rid of the openeye license

Reply:
I also expect this to skip it from being run from a fork 🤷‍♂️